includes/embeds/class-amp-twitter-embed.php

```diff
@@ -28,6 +28,13 @@ class AMP_Twitter_Embed_Handler extends AMP_Base_Embed_Handler {
 	 */
 	const URL_PATTERN_TIMELINE = '#https?:\/\/twitter\.com(?:\/\#\!\/|\/)(?P<username>[a-zA-Z0-9_]{1,20})(?:$|\/(?P<type>likes|lists)(\/(?P<id>[a-zA-Z0-9_-]+))?)#i';
 
+	/**
+	 * The tag (name) of the shortcode.
+	 *
+	 * @var string
+	 */
+	const SHORTCODE_TAG = 'vimeo';
```

Comment:
This should probably be `tweet`.

Reply:
Ah, good point.

Reply:
Addressed in https://github.com/ampproject/amp-wp/pull/3678/commits/ec5257ee50907b44281e8405918aa2fe1548aaf6. It turns out that the constant wasn't needed at all 😄